### ISSUE TITLE
replace os-package-registry with simillar but fitting datahub needs

### ISSUE THREAD

**Issue by zelima** [datahq/metastore#2]
os conductor was using [os-package-registry](https://github.com/openspending/os-package-registry) lib for communicating with elasticsearch. We need to create similar but fitting datahub needs 

## Acceptance criteria

* [ ] Reuse all need functionalities from `os-package-registry`
* [ ] fits our needs (schema design, etc..)

# Tasks

* [ ] create new repo ???
* [ ] update `os-package-registry` code fitting datahub needs ( #1 needs to be done first)

**Comment by zelima** [comment]
WONTFIX
